### DISCUSSION TITLE
Help Installing on Fedora Atomic (Aurora, Universal Blue Kinoite)

### DISCUSSION THREAD

**Discussion by RebelLion1519** [winapps-org/winapps#340] [General]
Hi, I'm a beginner here, this could well be operator error, but I didn't see anyone with this question already. I'm trying to install winapps on Aurora, which is an atomic build of Fedora (rpm-ostree based). 

1. Successfully created Windows VM in Podman
2. freerdp was already installed, version freerdp-2:3.9.0-1.fc41.x86_64. When I first had the error below, tried using the Flatpak of FreeRDP, but get the same issue...
3. Successfully created a WinApps Configuration File at ~/.config/winapps/winapps.conf with the correct user and password for my VM.
4. When running "bash <(curl https://raw.githubusercontent.com/winapps-org/winapps/main/setup.sh)" I get:

ERROR: MISSING DEPENDENCIES.
Please install 'dialog' to proceed.

So I ran:

"sudo rpm-ostree install dialog"

It seems to have installed (now running it gets "error: Package/capability 'dialog' is already requested") but I continue to get the same error when attempting to run the bash script for the installer. 

I don't know where to go from here. Thanks in advance for any assistance!

**Comment by eylenburg** [comment]
Did you restart your system after installing "dialog"? Package changes (with rpm-ostree) are only applied after a restart.

**Comment by RebelLion1519** [comment]
I thought I had, I don't remember. It wasn't meant to be. I also tried on Ubuntu, got a different error at about the same step, gave up and just installed Windows bare metal for now. I don't have a spare machine to play with currently, but when I do I'll revisit trying to get this to work. Thanks for the response.

-------- Original Message --------
